esx_service/cli/vmdkops_admin.py

```diff
@@ -1323,7 +1331,17 @@ def config_rm(args):
             info = auth.get_info()
             mode = auth.mode # for usage outside of the 'with'
         except auth_data.DbAccessError as ex:
-            return err_out(str(ex))
+            # the DB is broken and is being asked to be removed, so let's oblige
+            print("Received error - removing comfiguration anyways. Err: \"{}\"".format(str(ex)))
+            try:
+                os.remove(auth_data.AUTH_DB_PATH)
+            except:
+                pass
+        return None
```

Comment:
Shouldn't this return statement be inside the above "except auth_data.DbAccessError as ex:" block? The current indention seems incorrect as it will return in any case.